### PR TITLE
Mergesort c

### PR DESCRIPTION
**Pull request by Pratap2018**
## Have you read the Contribution Guidelines?

Yes

## Description

Merge sort in C

## Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)

Fixes Issue #5

### CODE REVIEW

File: C/Mergesort.c [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        scanf("%d",&arr[i]);
+    }
+    mergeSort(arr,0,n-1);
+    printf("Sorted Array :\t");
+    for(i=0;i<n;i++){
+        printf("%d ",arr[i]);
+    }
+
+
+}
```
Comment (sukritishah15) [inline C/Mergesort.c:77]: Add Sample I/P, O/P, Time and Space complexity in a single MULTILINE COMMENT.
Let me know, once done

Comment (Pratap2018) [reply C/Mergesort.c]: I have done that can I create new PR now?

Comment (sukritishah15) [reply C/Mergesort.c]: Yes, create a new PR from a new branch.
Closing this for now.

Comment (Pratap2018) [reply C/Mergesort.c]: I have created Pull request

**Pull request closed without merging**